src/Microsoft.OData.Core/ODataContextUrlInfo.cs

```diff
@@ -403,8 +403,9 @@ private static string CreateSelectExpandContextUriSegment(SelectExpandClause sel
         /// <returns>The generated expand string.</returns>
         private static string ProcessSubExpand(string expandNode, string subExpand, ODataVersion version)
         {
-            return string.IsNullOrEmpty(subExpand) && version <= ODataVersion.V4 ? null :
-                expandNode + ODataConstants.ContextUriProjectionStart + subExpand + ODataConstants.ContextUriProjectionEnd;
+
+            // TODO: cleanup the version parameter.

```

Comment:
>// TODO: cleanup the version parameter. [](start = 12, length = 39)

what work is left? not passing the version parameter to the method? do you have an issue filed for that?

Reply:
Yes, it is just potential cleanups from caller side, wherever applicable, after reviewing changes in this PR and confirming that the dependency on version (V4/V401) should be removed. The cleanup should be straight forward when this PR is finalized.